digester/Dockerfile

```diff
@@ -12,4 +14,4 @@ RUN go build -o app ./main.go
 
 EXPOSE 8080
 
-CMD ["./app"]
\ No newline at end of file
+CMD /wait && ./app
```

Comment:
What does it wait for?

Reply:
This: https://github.com/kossiitkgp/top-secret-stuff/blob/a9a067cffd1101d961323e809127466c621b5579/docker-compose-digester.yml#L10

Reply:
Interesting.